lib/resources.js

```diff
@@ -311,18 +318,21 @@ async function _getAcquisitionRecord({acquirerId}) {
         tokenized: [{
           tokenizerId,
           resources: {}
-        }]
+        }],
+        expires: new Date(),
```

Comment:
I'm confused about this.  Why is this being sent to now?  Seems to me anyone looking at this record downstream is going to think it's expired already.  Am I misreading this?

Reply:
No, you're not misreading it. This is the default record -- and it should expire immediately if nothing gets added to it. Setting it to any other time doesn't make any sense, would be arbitrary, and could cause it to unnecessarily persist.

Reply:
@dlongley you've done a great job documenting other aspects of this code.  If some commentary could be added here as well, that would be great.

Reply:
Added comments.